fabfile/adage_server.py

```diff
@@ -190,13 +186,38 @@ def import_data_and_index():
     run(('python manage.py genes_load_gene_history %s 208964 '
         '--tax_id_col=1 --discontinued_id_col=3 --discontinued_symbol_col=4') %
         CONFIG['data']['gene_history_file'])
-    run('python manage.py import_gene_network  %s "Ensemble ADAGE 300"' %
-        CONFIG['data']['gene_network_file'])
-    # Note that the ParticipationType "High-weight genes" has been
-    # created in: migrations/0009_auto_20170503_1700.py
-    run('python manage.py import_node_gene_network %s "Ensemble ADAGE 300"'
-        ' "High-weight genes"' % CONFIG['data']['node_gene_network_file'])
-    run('python manage.py import_gene_sample_expr %s 208964' %
+
+    mlmodel_basic = {
+        "title": "Ensemble ADAGE 300",
+        "cutoff": 0.4
+    }
+    mlmodel_complex = {
+        "title": "Ensemble ADAGE 300 with more complex gene-gene network",
+        "cutoff": 0.2
+    }
+    # Unzip gene-gene network file.
+    run('gunzip "%s.gz"' % CONFIG['data']['gene_network_file'])
```

Comment:
I don't know how I feel about the `config.py.template` file stating that it is not a gzipped file and then this manually tacking on the ".gz" ending. Should there be a way to check here if this is a gzipped file?